package.json

```diff
@@ -1,12 +1,16 @@
 {
   "name": "@good-fences/api",
-  "version": "0.1.2",
+  "version": "0.1.4",
```

Comment:
Doesn't this automatically get updated by beachball? Or is this to account for the bad published versions on npm rn?

Reply:
It's updated after it gets published, still hasn't been updated because of the publish part failing